api/envoy/service/metrics/v2/metrics_service.proto

```diff
@@ -5,7 +5,6 @@ package envoy.service.metrics.v2;
 option java_outer_classname = "MetricsServiceProto";
 option java_multiple_files = true;
 option java_package = "io.envoyproxy.envoy.service.metrics.v2";
-option go_package = "v2";
```

Comment:
Should `check_format` also look for these?

Reply:
Yes, good idea. go_package is actually evil since we have two sets of gogo/goproto files that cannot be same package.

Reply:
Added a check